integration/plugin/authz/authz_plugin_test.go

```diff
@@ -104,6 +104,13 @@ func TestAuthZPluginAllowRequest(t *testing.T) {
 }
 
 func TestAuthZPluginTLS(t *testing.T) {
+	// FIXME: TestAuthZPluginTLS feature depends on deprecated / legacy Common Name field
+	// with old certs:
+	//     authz_plugin_test.go:132: assertion failed: error is not nil: error during connect: Get "https://localhost:4271/v1.41/version": x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
+	//
+	// with updated certs:
+	//     authz_plugin_test.go:134: assertion failed: client (string) !=  (ctrl.reqUser string)
+	t.Skip("FIXME: this feature depends on deprecated / legacy Common Name field")
```

Comment:
Need to dig further why it doesn't pick up the CommonName field here. This should be set here; https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/pkg/authorization/middleware.go#L65-L74

```go
// Default authorization using existing TLS connection credentials
// FIXME: Non trivial authorization mechanisms (such as advanced certificate validations, kerberos support
// and ldap) will be extracted using AuthN feature, which is tracked under:
// https://github.com/docker/docker/pull/20883
if r.TLS != nil && len(r.TLS.PeerCertificates) > 0 {
    user = r.TLS.PeerCertificates[0].Subject.CommonName
    userAuthNMethod = "TLS"
}

authCtx := NewCtx(plugins, user, userAuthNMethod, r.Method, r.RequestURI)
```

But (I think) depends on this; https://github.com/golang/go/blob/2ebe77a2fda1ee9ff6fd9a3e08933ad1ebaea039/src/crypto/x509/pkix/pkix.go#L144-L194